maxwell_slice/sphere_scat.py

```diff
@@ -1,10 +1,11 @@
-from ._temporarydirectory import TemporaryDirectory
-from ._shellscript import ShellScript
+from _temporarydirectory import TemporaryDirectory
```

Comment:
I think we should make an examples folder and put sphere_scat.py as `examples/sphere_scat_example.py`